### ISSUE TITLE
Allow using structure marker textures on non-surveyor servers

### ISSUE THREAD

**Issue by Hyporeal** [sleepingdragoninn/antique-atlas#171]
I would prefer to have structures not show at all unless you've entered them. Alternatively, they could not show up at all and can be manually added. This would prevent situations where players would have never noticed a town or structure if they had not looked at the Atlas.

**Comment by sleepingdragoninn** [comment]
already possible! in config/surveyor.toml, set "structures" to DISABLED.

let me know if that works for you!

**Comment by Hyporeal** [comment]
Wow. That was an epically fast response! That would certainly help! I know there are custom markers, but it would also be helpful to have access to structural markers like towns, monuments, etc, for manual placement if they are otherwise disabled!

Thank you!

**Comment by sleepingdragoninn** [comment]
good suggestion - we'll see if we can surface an option to ignore marker filtering on the atlas side

**Comment by sleepingdragoninn** [comment]
Yep so i'm thinking by default this can be set up so that the structure markers appear as custom placeable markers when atlas is running in client mode (i.e surveyor is not installed on the server) - and that'll just be everything under the structure/ subfolder.

Then the config will let you force it on or off from there.

**Comment by Vritra22189** [comment]
any reason to only allowing structure markers on non-surveyor servers?  

i saw there is no plan to support auto place mod structures, 
but i think it still make sense for players to manually place structure markers to e.g. YUNG's, 
which are replacing vanilla structures.
